MCP_Server/server.py

```
@@ -183,478 +40,61 @@ async def server_lifespan(server: FastMCP) -> AsyncIterator[Dict[str, Any]]:
+    """Get or create a persistent Ableton connection."""
     global _ableton_connection
-    
     if _ableton_connection is not None:
         try:
-            # Test the connection with a simple ping
-            # We'll try to send an empty message, which should fail if the connection is dead
-            # but won't affect Ableton if it's alive
             _ableton_connection.sock.settimeout(1.0)
-            _ableton_connection.sock.sendall(b'')
+            _ableton_connection.sock.sendall(b"")
             return _ableton_connection
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)